apps/pgsql/patroni/openshift/deployment.yaml

```diff
@@ -229,7 +239,7 @@ objects:
           volume.beta.kubernetes.io/storage-class: ${STORAGE_CLASS}
         labels:
           app: ${NAME}${SUFFIX}
-        name: postgresql
+        name: ${NAME}${SUFFIX}
```

Comment:
PVCs created by `volumeClaimTemplates` will already have the statefulset name appended to the PVC name. We changed to `postgresql` as to keep the name shorter. It was long and repetitive. It seems like volumes defined in `statefulset.spec.volumeClaimTemplates` are implied `statefulset.spec.template.spec.volumes`, so we may not need to repeat.

- https://stackoverflow.com/questions/46442238/can-i-rely-on-volumeclaimtemplates-naming-convention

Reply:
Totally right, that was doubling everything up